src/tests/ftest/nvme/enospace.py

```diff
@@ -0,0 +1,440 @@
+                self.fail("One of the Background IOR job failed")
+
+    @skipForTicket("DAOS-4846")
+    def test_enospace_lazy_with_bg(self):
+        """Jira ID: DAOS-4756.
+
+        Test Description: IO gets DER_NOSPACE when SCM and NVMe is full with
+                          default (lazy) Aggregation mode.
+
+        Use Case: This tests will create the pool and fill 75% of SCM size which
+                  will trigger the aggregation because of space pressure,
+                  next fill 75% more which should fill NVMe. Try to fill 60%
+                  more and now SCM size will be full too.
+                  verify that last IO fails with DER_NOSPACE and SCM/NVMe pool
+                  capacity is full.
+
+        :avocado: tags=all,hw,medium,nvme,ib2,full_regression
+        :avocado: tags=der_enospace,enospc_lazy,enospc_lazy_bg
+        """
+        print(self.pool.pool_percentage_used())
+
+        #Run IOR to fill the pool.
+        self.run_enospace_with_bg_job()
+
+    @skipForTicket("DAOS-4846")
+    def test_enospace_lazy_with_fg(self):
+        """Jira ID: DAOS-4756.
+
+        Test Description: Fill up the system (default aggregation mode) and
+                          delete all containers in loop, which should release
+                          the space.
+
+        Use Case: This tests will create the pool and fill 75% of SCM size which
+                  will trigger the aggregation because of space pressure,
+                  next fill 75% more which should fill NVMe. Try to fill 60%
+                  more and now SCM size will be full too.
+                  verify that last IO fails with DER_NOSPACE and SCM/NVMe pool
+                  capacity is full. Delete all the containers.
+                  Do this in loop for 10 times and verify space is released.
+
+        :avocado: tags=all,hw,medium,nvme,ib2,full_regression
+        :avocado: tags=der_enospace,enospc_lazy,enospc_lazy_fg
+        """
+        print(self.pool.pool_percentage_used())
+
+        #Repeat the test in loop.
+        for _loop in range(10):
+            #Run IOR to fill the pool.
+            self.run_enospace_foreground()
+            #Delete all the containers
+            self.delete_all_containers()
+
+    @skipForTicket("DAOS-4846")
+    def test_enospace_time_with_bg(self):
+        """Jira ID: DAOS-4756.
+
+        Test Description: IO gets DER_NOSPACE when SCM is full and it release
+                          the size when container destroy with Aggregation
+                          set on time mode.
+
+        Use Case: This tests will create the pool. Set Aggregation mode to Time.
+                  Fill 75% of SCM size which will trigger the aggregation
+                  because mode set to time, next fill 75% more which will fill
+                  up NVMe. Try to fill 60% more and now SCM size will be full
+                  too. Verify last IO fails with DER_NOSPACE and SCM/NVMe
+                  pool capacity is full.

```

Comment:
In time mode, aggregation will be triggered from time to time, not related to space pressure and io activities.

Reply:
right, I will change the use case description.